inst/include/Rcpp/DataFrame.h

```diff
@@ -83,6 +83,26 @@ namespace Rcpp{
             return LENGTH(rn);
         }
 
+        // Leave in place for a year
```

Comment:
I have no interest / time / energy to chase *twenty* package maintainers over essentially nothing.

I have tried to get _five_ to change _an error of theirs_ for three weeks now and only got three.  No, we are not going to break that interface.

Additional accessors are perfectly fine.